src/examples/track_matching/reco_particles_track_matching.cc

```diff
@@ -14,7 +14,7 @@
 
 using std::string;
 
-void reco_particles_track_matching(char *file_name) {
+void reco_particles_track_matching(TString file_name) {
```

Comment:
```suggestion
void reco_particles_track_matching(std::string file_name) {
```

Reply:
`std::string` throws
```
error: no matching function for call to ‘TFile::TFile(std::string&)
```
Either `TString` or `const char*` is adequate.